src/store/index.js

```diff
@@ -0,0 +1,37 @@
+import { applyMiddleware, compose, createStore,  } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import { logger } from 'redux-logger';
+import promiseMiddleware from 'redux-promise-middleware';
+import thunk from 'redux-thunk';
+
+import reducers from '../reducers';
+
+const middleware = [
+  logger,
+  promiseMiddleware(),
+  thunk,
+];
+
+
+let composeEnhancers = process.env.NODE_ENV === 'development' ?
+  composeWithDevTools({}) :
+  compose;
+
+const store = createStore(
+  reducers,
+  composeEnhancers( applyMiddleware(...middleware))
+);
+
+export default store;
+
+/*
+ TODO: move to a test file
```

Comment:
Indeed, move into another file if you use the code for testing. Such commented out code should not live inside this repo.